roles/kubernetes/tasks/master.yml

```diff
@@ -53,12 +53,12 @@
         --upload-certs \
         --pod-network-cidr={{ k8s_network_cidr }}
       become: true
-      when: k8s_multi_master|int > 1
+      when: k8s_multi_master
```

Comment:
Remember that in ansible when you want to check a boolean best practices suggest:
```yaml
when: k8s_multi_master | bool
```